spinalcordtoolbox/scripts/sct_apply_transfo.py

```diff
@@ -328,7 +328,7 @@ def apply(self):
                     img_out = cropper.crop()
                 img_out.save(fname_out)
 
-        display_viewer_syntax([fname_dest, fname_out], verbose=verbose)
+        return fname_dest, fname_out
```

Comment:
Ah! Thank you for catching this. This was definitely a slip up on my part, rather than intentional. 

I've removed the `return` statement, as suggested. :) 

> (Incidentally, I don't see any benefit to `Transform` being a class rather than a function, but simplifying that is certainly out of scope for this PR. It's just a kind of superfluous object-oriented style that I've seen in a few places in SCT, something to be aware of.)

I had noticed this too, and very much agree here!